containers/Dockerfile.buildenv

```diff
@@ -37,6 +37,20 @@ RUN apt-get update -y && apt-get install -y libc++-dev libc++1 libc++abi-dev \
 RUN apt-get install -y ccache doxygen python-pip && pip install coverxygen \
     && pip install numpy
 
+# Install bibtex for Doxygen bibliography management
+# We first install the TeXLive infrastructure according to the configuration in
+# support/TeXLive/texlive.profile and then use it to install the bibtex package.
+RUN mkdir /work/texlive
+WORKDIR /work/texlive
+RUN wget http://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz \
+    && tar -xzf install-tl-unx.tar.gz \
+    && rm install-tl-unx.tar.gz \
+    && wget https://raw.githubusercontent.com/sxs-collaboration/spectre/develop/support/TeXLive/texlive.profile \
+    && install-tl-*/install-tl -profile=texlive.profile \
+    && echo "export PATH=\$PATH:/work/texlive/bin/x86_64-linux" >> /root/.bashrc \
```

Comment:
Is it not possible to run `nm -r install-tl-*` after to clean up the install-tl directory? This worked on my system but I didn't try rerunning Doxygen, etc. After I did that the installed texlive 17MB, which I think is completely reasonable :)

Could you move this block right to the end? That way if we want to update the latex packages we don't need to rebuild everything. That'll save between 1 and 2 hours per build.